sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -137,6 +137,17 @@ public void testDataSourceConfigurationDriverAndUrl() throws Exception {
   }
 
   @Test
+  public void testDataSourceConfigurationUsername() throws Exception {
+    JdbcIO.DataSourceConfiguration config = JdbcIO.DataSourceConfiguration.create(
+        "org.apache.derby.jdbc.ClientDriver",
+        "jdbc:derby://localhost:1527/target/beam",
+        "sa", "sa");
```

Comment:
This test is a new one that to illustrate the change. It fails without the current change on the `JdbcIO`.